src/locales/index.js

```diff
@@ -18,20 +18,60 @@ const mobileTranslations = {
   es: esMobile
 }
 
-const unsupportedLocaleTag = () => {
+const defaultLanguage = () => {
+  const polyglot = new Polyglot({onMissingKey: () => null})
+  return extendPolyglot(defaultLocaleTag, polyglot, availableTransations[defaultLocaleTag], mobileTranslations[defaultLocaleTag] )
+}
+
+
+const extendPolyglot = (locale, polyglot, phrases, mobilePhrases) => {
```

Comment:
This function interface isn't the easiest to understand at the call site. I'd recommend, with this many arguments, moving to an object interface with destructuring.